### DISCUSSION TITLE
Deploy Front End to CDN

### DISCUSSION THREAD

**Discussion by austincollinpena** [livebud/bud#25] [Ideas]
Is it possible to build a static site and deploy it to a CDN? Or would it be better to just build and cache certain directories?

**Comment by matthewmueller** [comment]
Yes, but it's currently _very hacky_. I do this with the welcome page: https://github.com/livebud/welcome

Check out the Readme for instructions. 

I'm planning to make this a lot nicer soonish. Once I fix #5 and finish up custom command support, you'll be able to `go get` a custom "export" command that may use staticgen under the hood, but tailored to Bud.
